### PR TITLE
Use Panda CSS

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/5868030762).

This PR adds the use of Panda CSS, with a vision for this replacing emotion entirely with the removal of `@stacks/ui`.

I have to use the export lint override as these assets are used in the config and that's not detected.

### CODE REVIEW

File: src/app/components/drawer/base-drawer.tsx [hunk 1]
```diff
@@ -101,10 +101,13 @@ export const BaseDrawer = memo((props: BaseDrawerProps) => {
...
         <Box
-          css={css`
-            overflow-y: scroll;
-            ${hideScrollbarStyle}
-          `}
+          className={
+            css({
+              overflowY: 'scroll',
+            }) +
+            ' ' +
```
Comment (pete-watters) [inline src/app/components/drawer/base-drawer.tsx:108]: Curious if this `' ' +` is needed or is that just prettier?

File: src/app/components/loading-rectangle.tsx [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
-  }
-  100% {
-    background-position: 500px;
-  }
-`;
 export function LoadingRectangle(props: BoxProps) {
   return (
     <Box
       backgroundImage="linear-gradient(90deg, rgba(219,219,219,1) 0%, rgba(192,192,247,0.5) 35%, rgba(219,219,219,1) 100%)"
       backgroundColor="rgb(219,219,219)"
```
Comment (pete-watters) [inline src/app/components/loading-rectangle.tsx:7]: Is this just working using `shine`? Pretty clean if so

File: theme/colors.ts
Comment (pete-watters) [inline theme/colors.ts:1]: 👏

File: theme/keyframes.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { CssKeyframes } from 'leaf-styles/types/system-types';
+
```
Comment (pete-watters) [inline theme/keyframes.ts:2]: Are these really just not used yet? Or is there some magic in Panda

**Review by pete-watters — APPROVED** [review]
Nice work 👍 it's seems to integrate relatively easily

**Review by alter-eggo — APPROVED** [review]

**Review by fbwoolf — APPROVED** [review]

File: src/app/components/drawer/base-drawer.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/app/components/drawer/base-drawer.tsx:108]: it's ugly but basically a hack as css just returns classnames we wanna add many, could also make array and join. Happy to change this

File: theme/keyframes.ts (hunk 3 above)
Comment (kyranjamie) [reply theme/keyframes.ts:2]: Can probs adjust the linter to avoid the need

File: src/app/components/drawer/base-drawer.tsx (hunk 1 above)
Comment (pete-watters) [reply src/app/components/drawer/base-drawer.tsx:108]: ah yes, now I remember this, thanks! this is fine, I was just wondering. Ugly and working is fine, I can relate!

**Pull request merged**